spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -111,6 +111,9 @@
         expect(@analytics).to receive(:track_mfa_submit_event).
           with(properties)
 
+        expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:mfa_phone_verification_otp_submitted).
+          with({ success: false })
+
```

Comment:
The approach is good, but I think we want to follow the new `stub_analytics_events` + `expect(@irs_attempts_api_tracker)...` pattern here.